zookeeper-server/src/main/java/org/apache/zookeeper/common/IOUtils.java

```diff
@@ -68,6 +68,7 @@ public static void cleanup(Logger log, Closeable... closeables) {
 
     /**
      * Copies from one stream to another.
+     * Deprecated. No longer used in ZK.
```

Comment:
If it's deprecated, it should use the `@Deprecated` annotation, and have a useful `@deprecated` javadoc message for any current callers.

Reply:
To get past the build problems, just suppress the warning wherever it must still be used internally... or replace internal calls with whatever is replacing this.